app/views/organisations/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <% page_title @organisation.name %>
 <% page_class "organisations-show" %>
 <% atom_discovery_link_tag atom_feed_url_for(@organisation), "Latest activity" %>
-<%= api_link_tag api_organisation_url(@organisation) %>
```

Comment:
Isn't this still relevant? The public URL should be the same, maybe the magic rails function could be replaced with one that just generates the right URL?

Reply:
That view's not actually rendered any more - the live one is [over in collections](https://github.com/alphagov/collections/blob/master/app/views/organisations/_meta.html.erb#L5). Removing all the org page stuff in its entirety is a larger task

Reply:
Right, ok. I was thinking that maybe this was still used for some pages, but I think I'm actually thinking of some other type of content... :+1: